browser/views.py

```diff
@@ -624,3 +624,71 @@ def attendee_create(request, course_id):
         'form': form,
     }
     return render(request, template, context)
+
+
+@staff_member_required
+def add_investigator_record(request, person_id):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/investigator_create.html"
+    form = InvestigationForm()
+
+    if request.method == 'POST':
+        if 'SaveButton_Clicked' in request.POST:
```

Comment:
why do you ask for ```SaveButton_Clicked``` to be present in the post request?

Reply:
Removed SaveButton_Clicked from post request , implemented the functionality through JQuery